pixi.toml

```
@@ -11,9 +11,9 @@ postinstall = "pip install --no-build-isolation --no-deps --disable-pip-version-
 python = ">=3.10"
 rust = "=1.85"
 
+fsspec = "*"
```

```suggestion
fsspec = ">=2025.9"
```
they changed between semver and calver at some point.